dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/#python"
+COPYRIGHT="The D-Bus maintainers"
```

Comment:
The proper format for this is: `<Year range> <Copyright holder>`. Usually the best way to get this information is to dig straight into the source, but luckily the folk at Debian does this already :) https://metadata.ftp-master.debian.org/changelogs/main/d/dbus-python/dbus-python_1.2.14-1_copyright

Please import the list of copyright holders as well as licenses.

Reply:
The license was incorrectly represented in the `pypi` page of the Python packaged. Will be fixed.